go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/cpuguy83/go-md2man v1.0.10
 	github.com/cri-o/ocicni v0.1.1-0.20200218164758-d2881573038f
 	github.com/cyphar/filepath-securejoin v0.2.2
+	github.com/docker/docker v1.4.2-0.20191219165747-a9416c67da9f
```

Comment:
Do we have any alternative to this dependency?

Reply:
good catch, fixing